CODEOWNERS

```diff
@@ -4,6 +4,7 @@
 /elements/pl-external-grader-variables @nicknytko @eliotwrobson
 /elements/pl-graph @nicknytko @eliotwrobson
 /elements/pl-hidden-hints @eliotwrobson
+/elements/pl-template @eliotwrobson
```

Comment:
Nit: alphabetical order please (so after the `/elements/pl-rich-text-editor` line).